keepassxc-browser/background/keepass.js

```diff
@@ -809,7 +809,7 @@ keepass.disableAutomaticReconnect = function() {
 keepass.reconnect = async function(tab, connectionTimeout) {
     keepassClient.connectToNative();
     keepass.generateNewKeyPair();
-    const keyChangeResult = await keepass.changePublicKeys(tab, true, connectionTimeout).catch(() => false);
+    const keyChangeResult = await keepass.changePublicKeys(tab, !!connectionTimeout, connectionTimeout).catch(() => false);
```

Comment:
I am definitely not in favor of using !! here. Use an actual comparitor.

Reply:
If you don't use `!!`, you have to manually check if it is null or undefined, which needs to be written as `typeof connectionTimeout === 'number' && !Number.isNaN(connectionTimeout)`. This appears quite ugly.

Reply:
Gotta brush up on my Javascript, but why would it not have a default time definition?

Reply:
Sometimes two parameters are passed when calling, and sometimes no parameters are passed.

https://github.com/keepassxreboot/keepassxc-browser/blob/b0fd415e425d9e37c7826e4c2904e6eaeae6fb4e/keepassxc-browser/background/init.js#L10

https://github.com/keepassxreboot/keepassxc-browser/blob/b0fd415e425d9e37c7826e4c2904e6eaeae6fb4e/keepassxc-browser/background/keepass.js#L798

Reply:
This is because we want to have increased timeout at the first connect, if the browser is starting a bit slow on older systems.

Reply:
If you really want to use the default value, I suggest explicitly declaring it by using `keepass.reconnect(null, keepassClient.messageTimeout)` instead of implicitly.